### PR TITLE
Scope Under Variable Name & Split Commands to Classes

### PR DESCRIPTION
**Pull request by RedVortexDev**
beeg refactor to replace the huge Commands class with separate classes for each command

### CODE REVIEW

**Comment by RedVortexDev** [comment]
aw dang it

**Comment by RedVortexDev** [comment]
accidentally messed up by bringing the commits from the other branch to here, i really cant be bothered to fix this.. sorry :/

File: src/main/java/dev/dfonline/codeclient/command/impl/CommandGetActionDump.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import dev.dfonline.codeclient.ChatType;
+import dev.dfonline.codeclient.CodeClient;
+import dev.dfonline.codeclient.Utility;
+import dev.dfonline.codeclient.action.impl.GetActionDump;
+import dev.dfonline.codeclient.command.Command;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class CommandGetActionDump extends Command {
```
Comment (GeorgeRNG) [inline src/main/java/dev/dfonline/codeclient/command/impl/CommandGetActionDump.java:13]: would this not be an action command?

File: src/main/java/dev/dfonline/codeclient/command/impl/CommandWorldPlot.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import dev.dfonline.codeclient.command.Command;
+import dev.dfonline.codeclient.location.Plot;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class CommandWorldPlot extends Command {
+    @Override
+    public String name() {
+        return "worldplot";
```
Comment (GeorgeRNG) [inline src/main/java/dev/dfonline/codeclient/command/impl/CommandWorldPlot.java:14]: I would add a more sane alias, `plotsize` or `setplotsize` or `updateplotsize`, maybe we could add cc to the name. it doesn't really matter how long it is.

**Review by GeorgeRNG — CHANGES_REQUESTED** [review]

File: src/main/java/dev/dfonline/codeclient/command/impl/CommandWorldPlot.java (hunk 2 above)
Comment (RedVortexDev) [reply src/main/java/dev/dfonline/codeclient/command/impl/CommandWorldPlot.java:14]: plotsize is good

**Pull request merged**